gravitee-apim-portal-webui-next/src/app/log-in/reset-password/reset-password-confirmation/reset-password-confirmation.component.html

```diff
@@ -0,0 +1,85 @@
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<mat-card appearance="outlined" class="reset-password-confirmation">
+  @if (!isSubmitted) {
+    <mat-card-header>
+      <mat-card-title i18n="@@resetPasswordConfirmationTitle">Reset password confirmation</mat-card-title>
+    </mat-card-header>
+    @if (!isTokenExpired && !!resetPasswordConfirmationForm) {
+      <mat-card-content>
+        <form class="reset-password-confirmation__form" [formGroup]="resetPasswordConfirmationForm" (ngSubmit)="confirmResetPassword()">
+          <div class="reset-password-confirmation__form__fields">
+            <mat-form-field class="reset-password-confirmation__form__input">
+              <mat-label i18n="@@resetPasswordConfirmationPassword">Password</mat-label>
+              <input matInput formControlName="password" type="password" appearance="outlined" />
+              @if (resetPasswordConfirmationForm.controls['password'].hasError('required')) {
+                <mat-error i18n="@@resetPasswordConfirmationPasswordErrorRequired">Password is required</mat-error>
+              }
+            </mat-form-field>
+            <mat-form-field subscriptSizing="dynamic" class="reset-password-confirmation__form__input">
+              <mat-label i18n="@@resetPasswordConfirmationConfirmedPassword">Confirm Password</mat-label>
+              <input matInput formControlName="confirmedPassword" type="password" appearance="outlined" />
+
+              @if (resetPasswordConfirmationForm.controls.confirmedPassword.hasError('required')) {
+                <mat-error i18n="@@resetPasswordConfirmationConfirmedPasswordErrorRequired">Confirm Password is required </mat-error>
+              } @else if (resetPasswordConfirmationForm.controls.confirmedPassword.hasError('passwordMismatch')) {
+                <mat-error class="reset-password-confirmation__form__input" i18n="@@resetPasswordConfirmationConfirmedPasswordErrorMismatch"
+                  >Passwords do not match
+                </mat-error>
+              }
+            </mat-form-field>
+
+            @if (error() !== 200) {
+              <mat-error i18n="@@resetPasswordFailed"
+                >Password reset failed. Please try again or contact your administrator for assistance.
+              </mat-error>
+            }
+          </div>
+          <div class="reset-password-confirmation__form__buttons">
+            <button
+              [disabled]="resetPasswordConfirmationForm.invalid"
+              type="submit"
+              mat-flat-button
+              i18n="@@resetPasswordConfirmationAction"
+              class="reset-password-confirmation__form__submit secondary-button">
+              Reset password
+            </button>
+          </div>
+        </form>
+      </mat-card-content>
+    } @else {
+      <mat-card-content class="reset-password-confirmation__expired">
```

Comment:
In another technical task it would be nice to introduce a generic card to avoid to duplicate this kind of "default" states